auto_submit/lib/service/config.dart

```diff
@@ -37,9 +37,13 @@ class Config {
   static const String kRevertLabel = 'revert';
 
   static const int backOfMultiplier = 200;
-  static const int backoffAttempts = 3;
+  static const int backoffAttempts = 5;
```

Comment:
It would be cleaner to consolidate these into a [RetryOption](https://pub.dev/documentation/retry/latest/retry/RetryOptions-class.html)

Reply:
That is a good idea.